src/internal/modules/billing/lib/mappers.js

```diff
@@ -41,23 +41,24 @@ const mapChargeElementTransactions = group => {
     netTotal: acc.netTotal + row.value
   }), initialValue);
 
+  const mappedTransactions = transactions.map(mapTransaction);
   return {
-    transactions: transactions.map(mapTransaction),
+    transactions: mappedTransactions.filter(txn => !txn.isMinimumCharge),
+    minimumChargeTransactions: mappedTransactions.filter(txn => txn.isMinimumCharge),
```

Comment:
Is there a UI requirement to put these at the bottom of the table?
Otherwise we wouldn't need to group them separately.